run_all_tests.sh

```diff
@@ -7,9 +7,6 @@
 # Fail immediately in case of errors and/or unset variables
 set -eu -o pipefail
 
-# Run all java tests and native unit tests.
-./run_maven_tests.sh
-
-# Run native integration tests that require prepared classpaths for fake classes.
-./run_native_integration_tests.sh --skip-compile
-./run_ejb_app_tests.sh
+echo "Start running all EJB tests"
```

Comment:
Is that correct that it doesn't run the light-client tests?

Reply:
yes. Because they will be added later, with LK module